package.json

```diff
@@ -206,8 +206,8 @@
     "watch": "tsc -watch -p ./"
   },
   "dependencies": {
-    "@thomaskoppelaar/markdown-it-wikilinks": "^1.3.0",
```

Comment:
Trying to understand other implications of this change. Your goal is to make it so the wiki links are clickable inside the vscode markdown preview -- but does this break any other use cases? Eg, is this going to break things somehow for html generated for other purposes / for viewing outside of the Markdown Prview?